build.ps1

```diff
@@ -24,12 +24,14 @@ Implement build and packaging of the package and place the output $OutDirectory/
 function DoBuild
 {
     Write-Verbose -Verbose "Starting DoBuild"
-    Get-ChildItem -Path $script:ModuleRoot -Filter "$script:ModuleName.ps*1" | ForEach-Object { Copy-Item -Path $_.FullName -Destination $script:OutModule -Verbose }
+    Get-ChildItem -Path $script:ModuleRoot -Filter "*.ps*1" | ForEach-Object { Copy-Item -Path $_.FullName -Destination $script:OutModule -Verbose }
+    Copy-Item -Path (Join-Path $script:ModuleRoot 'yml') -Recurse $script:OutModule -Force
+
     Write-Verbose -Verbose "Ending DoBuild"
 }
 
 #region Special casing for PSPackageProject CI system
-$PSPackageProjectModule = Join-Path $PSScriptRoot $SrcPath -AdditionalChildPath "$ModuleName.psd1"
+$PSPackageProjectModule = [System.IO.Path]::Join($PSScriptRoot, $SrcPath, "$ModuleName.psd1")
```

Comment:
`Path.Join` [isn't available in .NET Framework](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.join?view=netstandard-2.1&viewFallbackFrom=netframework-4.8#System_IO_Path_Join_System_String_System_String_System_String_)

Reply:
fixed